pkg/northbound/gnmi/get.go

```diff
@@ -216,3 +236,16 @@ func buildUpdate(prefix *gnmi.Path, path *gnmi.Path, configValues []*devicechang
 		Val:  value,
 	}, nil
 }
+
+func extractGetVersion(req *gnmi.GetRequest) (devicetype.Version, error) {
+	var version devicetype.Version
+	for _, ext := range req.GetExtension() {
+		if ext.GetRegisteredExt().GetId() == GnmiExtensionVersion {
+			version = devicetype.Version(ext.GetRegisteredExt().GetMsg())
+		} else {
+			return "", status.Error(codes.InvalidArgument, fmt.Errorf("unexpected extension %d = '%s' in Set()",
```

Comment:
Comment should say in `Get()`

Do we really want to throw error though?
We could be more forgiving in Get() and just log the unexpected extension